src/Test/Tasty/Wai.hs

```diff
@@ -58,7 +60,7 @@ instance IsTest Sess where
     -- 'Session a' isn't important for the test?
     E.try (runSession sess app) >>= either toFailure toPass
     where
-      toFailure (WaiTestFailure s) = testFailed <$> (formatMessage s)
+      toFailure (HUnitFailure _ s) = testFailed <$> (formatMessage (formatFailureReason s))
```

Comment:
Has the `WaiTestFailure` constructor disappeared?

Reply:
Yes, and the type it constructed.